commit 8d92f24852420c425f3ee499d3c3be01be4dc15c
Author: Eric Wong <normalperson@yhbt.net>
Date:   Wed Nov 21 00:57:33 2007 -0800

    t9106: fix a race condition that caused svn to miss modifications
    
    carbonated beverage noticed this test was occasionally failing.
    
    Signed-off-by: Eric Wong <normalperson@yhbt.net>

diff --git a/t/t9106-git-svn-dcommit-clobber-series.sh b/t/t9106-git-svn-dcommit-clobber-series.sh
index d59acc8..7452546 100755
--- a/t/t9106-git-svn-dcommit-clobber-series.sh
+++ b/t/t9106-git-svn-dcommit-clobber-series.sh
@@ -22,6 +22,7 @@ test_expect_success '(supposedly) non-conflicting change from SVN' "
 	cd tmp &&
 		perl -i -p -e 's/^58\$/5588/' file &&
 		perl -i -p -e 's/^61\$/6611/' file &&
+		poke file &&
 		test x\"\`sed -n -e 58p < file\`\" = x5588 &&
 		test x\"\`sed -n -e 61p < file\`\" = x6611 &&
 		svn commit -m '58 => 5588, 61 => 6611' &&
